testing/features/support/components/pagination/paging_info.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Pagination
+  class PagingInfo < ::Base
+    set_url "/iframe.html?id=components-pagination--paging-info&args=&viewMode=story"
+
+    element :number_of_results, ".cads-paging-info"
```

Comment:
You can avoid using styling classes in these selectors by using the element and text option, like this:
```
element :number_of_results, "h2", { text: "123456 results" }
```